notebooks/05-operations/01-logs-configuration.ipynb

```diff
@@ -71,17 +71,17 @@
     "cd learn-druid-local\n",
     "```\n",
     "\n",
-    "Pull and unpack a compatible version of Apache Druid.\n",
+    "Pull and unpack a compatible version of Apache Druid from the [Apache Druid downloads page](https://dlcdn.apache.org/druid/), for example `30.0.0`.\n",
     "\n",
     "```bash\n",
-    "wget https://dlcdn.apache.org/druid/30.0.0/apache-druid-30.0.0-bin.tar.gz\n",
-    "tar -xzf apache-druid-30.0.0-bin.tar.gz\n",
+    "wget https://dlcdn.apache.org/druid/<version>/apache-druid-<version>-bin.tar.gz\n",
```

Comment:
```suggestion
    "version='30.0.0'; wget https://dlcdn.apache.org/druid/$version/apache-druid-$version-bin.tar.gz\n",
```
could we write this so that the example works. same comment all notebooks

Reply:
Fixed across all three notebooks :)